standards/storing-source-code.md

```diff
@@ -49,3 +42,25 @@ By default, any [ministryofjustice organisation](https://github.com/ministryofju
 
 [template repository]: https://github.com/ministryofjustice/template-repository
 [github actions]: https://github.com/ministryofjustice/github-actions
+
+### GitHub user accounts
+
+If you already have a GitHub account from before you joined MOJ, you can choose whether to continue using it at MOJ or create a new one. Some people prefer to have continuity with previous work; others value the separation.
+
+You must [add your MOJ email address to your
+account](https://help.github.com/articles/adding-an-email-address-to-your-github-account/)
+It doesn't matter if it's the 'primary email' or not and it's ok to make it private.
+Maybe [use that email for notifications](https://help.github.com/articles/managing-notification-emails-for-organizations/)).
```

Comment:
This might be better as “You may want to […]”

Reply:
Now changed, exactly as suggested